packages/dx-react-scheduler-material-ui/src/templates/editing-menu/layout.jsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+export const Layout = React.memo(({
+  buttonComponent: Button,
+  handleClose,
+  commit,
+  availableOperations,
+  getMessage,
+  isDeleting,
+  ...restProps
+}) => {
+  const [currentValue, setCurrentValue] = React.useState(availableOperations[0].value);
+  const onCommitButtonClick = () => {
+    handleClose();
+    commit(currentValue);
+  };
+
+  const handleChange = (event) => {
```

Comment:
We can use `useCallback` to prevent rerender. Handle change doesn't depend on props